pkgs/tools/misc/tmux-xpanes/default.nix

```diff
@@ -0,0 +1,33 @@
+{ stdenv, fetchFromGitHub, openssl, perl }:
+
+stdenv.mkDerivation rec {
+  pname = "tmux-xpanes";
+  version = "4.1.0";
+
+  src = fetchFromGitHub {
+    owner = "greymd";
+    repo = "${pname}";
+    rev = "v${version}";
```

Comment:
no need to quote: repo = pname;

Reply:
thanks, I've also remove the quotes across the other variabels. Please have a look and let me know.

Reply:
fixed!